src/schema/sources.ts

```diff
@@ -1496,6 +1518,7 @@ export const resolvers: IResolvers<any, Context> = {
                 description,
                 memberPostingRank: sourceRoleRank[memberPostingRole],
                 memberInviteRank: sourceRoleRank[memberInviteRole],
+                private: isPrivate,
```

Comment:
If someone sends `private: null` this part will throw `UNEXPECTED` because `private` is non null column. Maybe extract the update payload at the top and then only set `private` on it if above `if` statement that you added passes.

Reply:
Yeah, thought at first it might become a bit of a change, but yes, that should be the right way to go. Will update quickly.

Reply:
Updated and included a test for it 🚀